### PR TITLE
DEL-31: Fix Javadoc Generation for 4.3.0

### PR DESCRIPTION
**Pull request by lvzh90**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -425,6 +426,65 @@
...
+                        <link>http://jakarta.apache.org/commons/lang/api-2.1/</link>
+                        <link>http://jakarta.apache.org/commons/pool/apidocs/</link>
+                        <link>http://logging.apache.org/log4j/2.x/log4j-api/apidocs/</link>
+                        <link>http://www.slf4j.org/api/</link>
+                        <link>http://junit.sourceforge.net/javadoc/</link>
+                    </links>
+                    <groups>
+                        <group>
+                            <title>Mule Core</title>
+                            <packages>org.mule.runtime.core.*</packages>
```
Comment (anosenzo) [inline pom.xml:468]: Linda why do we need this here?

File: pom.xml [hunk 2]
```diff
@@ -425,6 +426,65 @@
         </plugins>
     </build>
 
+    <reporting>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
```
Comment (anosenzo) [inline pom.xml:433]: why are you adding in the reporting section this?

File: pom.xml (hunk 1 above)
Comment (aiannucci) [reply pom.xml:468]: I think we could remove the groups section in these repos

Comment (lvzh90) [reply pom.xml:468]: Done

File: pom.xml (hunk 2 above)
Comment (lvzh90) [reply pom.xml:433]: It is necessary to generate the javadoc report

**Pull request closed without merging**